### PR TITLE
changelog changes

### PR DESCRIPTION
**Pull request by VarunSharma15**
resolves #

### Checklist
 - [ ] I have run this code in development and it appears to resolve the stated issue
 - [ ] This PR includes tests, or tests are not required/relevant for this PR
 - [ ] I have updated the `CHANGELOG.md` with information about my change

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 ## dbt-teradata 1.0.0a
 
 ### Features
+* dbt-teradata 1.6.7 is now compatible with dbt-core 1.6.7
 
 ### Fixes
-
+* Issues fixed related to cross DB macros ( https://github.com/Teradata/dbt-teradata/issues/105 , https://github.com/Teradata/dbt-teradata/issues/102)
```
Comment (SatishChGit) [inline CHANGELOG.md:7]: can you mention issues with numbering instead of between ()

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+* Reconfiguration of dbt-teradata snapshot code for fixing (https://github.com/Teradata/dbt-teradata/issues/65)
```
Comment (SatishChGit) [inline CHANGELOG.md:8]: Please add more description about the issue. Its not clear what we fixed

**Review by SatishChGit — CHANGES_REQUESTED** [review]
Couple of comments needs to be addressed

File: CHANGELOG.md (hunk 2 above)
Comment (VarunSharma15) [reply CHANGELOG.md]: Added some more context

File: CHANGELOG.md (hunk 1 above)
Comment (VarunSharma15) [reply CHANGELOG.md]: Done

**Review by SatishChGit — APPROVED** [review]

**Review by tallamohan — APPROVED** [review]
Looks Good

**Pull request merged**